web_external/vue/components/AnnotationTool/AnnotationToolFeatures.vue

```diff
@@ -0,0 +1,172 @@
+<template lang="pug">
+.annotation-tool-features
+  .alert.alert-warning.annotation-body-help
+    i.icon-help-circled
+    template(v-if='showReview')
+      span Mouse over a feature type to display markup.
+    template(v-else)
+      span(v-if='anyActive') Markup regions with this feature type.
+      span(v-else) Click a feature type to select it for markup.
+  .alert.alert-info.annotation-body-help
+    i.icon-info-circled
+    span(v-if='anyActive')
+      | #[b Left-click] to markup tiles.<br>
+      | #[b Shift-left-click] to erase tiles.<br>
+      | #[b Mouse wheel] to zoom in and out.<br>
+      | #[b Right-click and drag] to move while zoomed in.
+    span(v-else)
+      | #[b Mouse wheel] to zoom in and out.<br>
+      | #[b Left-click and drag] to move while zoomed in.<br>
+      | #[b Right-click and drag] to zoom in and out.<br>
+      | &nbsp;
+  div.annotation-section(v-show='!showReview')
+    label Confidence&nbsp;
+    .btn-group(data-toggle='buttons')
+      label.annotation-radio-btn.btn.btn-default(
+        :class='{active: markupState === 1.0}'
+      )
+        input(
+          type='radio',
+          name='certainty',
+          :value='1.0',
+          :checked='markupState === 1.0',
```

Comment:
What do you think about using a [Two-way Computed Property](https://vuex.vuejs.org/en/forms.html) here instead (or more generally)? It would have avoided the bug you fixed.

Reply:
These radios used v-model in my initial implementation--before Vuex--but I changed that at some point. Now that this uses Vuex, I updated this to use a two-way computed property. It's much cleaner. Thanks!

Reply:
@msmolens You can remove the `onCertaintyChanged` now too, right?